manifest.json

```diff
@@ -74,7 +74,22 @@
               "os": "linux",
               "osVersion": "alpine3.12",
               "tags": {
-                "alpine-3.12-helix-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
+                "alpine-3.12-helix-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "alpine-3.12-helix": {
+                  "isLocal": true
+                }
```

Comment:
This wouldn't be needed for the helix tag.  It should actually be defined for a `alpine-3.12` tag that you'd define for the new Dockerfile I mentioned that should be created.